src/CoreShop/Component/Payment/Model/Payment.php

```diff
@@ -24,7 +24,7 @@
 /**
  * @psalm-suppress MissingConstructor
  */
-class Payment implements PaymentInterface
+class Payment extends \Payum\Core\Model\Payment implements PaymentInterface
```

Comment:
needs to be done in Core\Model\Payment, Payment Component does not have a dependency to Payum.